package.json

```diff
@@ -26,7 +26,7 @@
   "license": "Apache-2.0",
   "dependencies": {
     "android-versions": "^1.7.0",
-    "cordova-common": "^4.0.2",
+    "cordova-common": "^4.0.2", 
```

Comment:
@liyamahendra Can you revert this accidental change?

Also rebase with master after. I have released cordova-common@5.0.0, bumped the package, and merged into master.

https://github.com/apache/cordova-android/pull/1566

Reply:
@erisu please review it again, when you can. Thanks

Reply:
Some reason the rebase didn't look correct.

If it is OK, I can try and rebase your branch to fix it.

Reply:
@liyamahendra sorry for the delayed response.

I had reverted and rebased the branch correctly. It no longer shows changes that doesnt belong in this PR.

You will most likely need to force pull the changes since it was a rebase.

I will review the changes again.

Reply:
If you can also review the commit changes one time to confirm nothing was lost, that will be great.

So far everything looks great on my end.

There is though a test failure

> ✗ Test#010 : Should detech adaptive icon with vector foreground and throws error for missing backwards compatability settings. (0.013 sec)
>         - Expected function to throw CordovaError: For the following icons with the density of: mdpi, adaptive foreground with a defined color or vector can not be used as a standard fallback icon for older Android devices. To support older Android environments, please provide a value for the src attribute., but it threw Error: ENOENT: no such file or directory, open '/home/runner/work/cordova-android/cordova-android/platforms/android/app/src/main/res/mipmap-mdpi-v26/ic_launcher.xml'.

It might be related to this change you made.

https://github.com/apache/cordova-android/pull/1550/files#diff-c6eb45ef369fbd224eff9d2927942c4732c0e8ffe2779a74e05aee9f15b5bdb2L308-R314

You changed the `.xml` file extension to `.png`... And the test is about `vector foreground`...

Reply:
@erisu you're right - changing the extension was causing the test to fail. Corrected that change. 

Please have a look and let me know if there is anything else.